### PR TITLE
License header to all cs files

### PR DESCRIPTION
**Pull request by isaya1910**

### CODE REVIEW

**Review by dtrabo — DISMISSED** [review]

File: AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
```
Comment (guperrot) [inline AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java]: Why adding a blank line? On other repos this is not what we did. Did you use a tool that does that?
Do we have only 15 files on Unity that needs licence?

**Review by guperrot — CHANGES_REQUESTED** [review]

File: AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java (hunk 1 above)
Comment (MatkovIvan) [reply AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java]: A lot of files already have this header 
https://github.com/Microsoft/AppCenter-SDK-Unity/blob/develop/Assets/AppCenter/AppCenterBehavior.cs#L3

Comment (MatkovIvan) [reply AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java]: It looks like a blank line copied from existing file headers

Comment (guperrot) [reply AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/StartupType.java]: Lets remove that blank line then.

File: Assets/AppCenter/Plugins/Android/appcenter-analytics-release.aar.meta [hunk 2]
```diff
@@ -1,33 +0,0 @@
-fileFormatVersion: 2
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/Android/appcenter-analytics-release.aar.meta]: Please revert

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by guperrot — APPROVED** [review]

**Pull request merged**